ocw/lib/cleanup.py

```diff
@@ -47,6 +47,23 @@ def list_clusters():
                     type(e).__name__, vault_namespace), traceback.format_exc())
 
 
+def delete_snapshots():
+    cfg = ConfigFile()
+    if cfg.has('snapshots'):
+        for vault_namespace in cfg.getList(['snapshots', 'namespaces'], ['']):
+            try:
+                delete_after_days = date.today() - timedelta(days=cfg.getInt(['snapshots', 'delete_after_days'], 30))
```

Comment:
In general I would put this configuration into ´[cleanup]` section.
My proposal would be:
```
[cleanup]
ec2-max-snapshot-age-days:
```


And till now, we tried to keep the configurations of such times in a global and namespace specific view.
So a configuration like
```
[snapshots]
namespaces=qam, kernel-qa
delete_after_days = 30
[snapshots.namespace.qam]
delete_after_days = 60
```
should work and have different settings for `qam` and `kernel-qa`. see https://github.com/cfconrad/pcw/blob/d063300e4ecc5d87e9513aefd787f043da417345/ocw/lib/provider.py#L14

Reply:
I am not sure that I can follow you here . 
For me this 
```
 [cleanup]
 ec2-max-snapshot-age-days:
```

is contradicting with this : 
```
[snapshots]
namespaces=qam, kernel-qa
delete_after_days = 30
[snapshots.namespace.qam]
delete_after_days = 60
```

anyway your comment push me to little bit different idea but which might make you happy . Can you please check latest code ?